python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceConversationalRemoteInferencePromptSchema.ts

```diff
@@ -0,0 +1,23 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const HuggingFaceConversationalRemoteInferencePromptSchema: PromptSchema = {
+  // See API Ref: https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/inference/_client.py#L338
+  // See Docs ref: https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.conversational
+  // for supported params.
+  // The settings below are supported settings specified in the HuggingFaceConversationalRemoteInference
+  // refine_completion_params implementation.
+  input: {
+    type: "string",
+  },
+  model_settings: {
+    type: "object",
+    properties: {
```

Comment:
What about 'parameters' property? Looks like that's supported according to

https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.conversational

Also I'm guessing we will switch to OpenAI schema with their latest update?

Reply:
Actually I read some more and looks like there are a lot of options supported for conversational task. Check out the conversational task here: https://huggingface.co/docs/api-inference/detailed_parameters

Those properties should be supported

Reply:
will add in the properties linked in a diff on top. Shipped a basic version of the model parser to test/validate real world test

Reply:
> Also I'm guessing we will switch to OpenAI schema with their latest update?


Yes. created #1232